README.md

```diff
@@ -1,16 +1,12 @@
-[![Build Status](https://travis-ci.org/kRITZCREEK/purescript-psc-ide.svg?branch=master)](https://travis-ci.org/kRITZCREEK/purescript-psc-ide)
-
-purescript-psc-ide
-===
+# purescript-psc-ide [![Build Status](https://travis-ci.org/kRITZCREEK/purescript-psc-ide.svg?branch=master)](https://travis-ci.org/kRITZCREEK/purescript-psc-ide)
```

Comment:
Updated the Markdown file according to markdownlint's rules, but I can revert these changes if you'd like. Just thought I'd do it while I'm already linting things.